website/docs/r/cis_origin_auth.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Internet services"
+layout: "ibm"
+page_title: "IBM: ibm_cis_origin_auth"
+description: |-
+  Provides an IBM CIS origin auth resource.
+---
+
+# ibm_cis_origin_auth
+ Provides origin auth settings resource. The resource allows to create, update, or delete cache settings of a domain of an IBM Cloud Internet Services CIS instance. For more information about CIS origin auth, see [CIS ORIGIN AUTH](https://cloud.ibm.com/docs/cis?topic=cis-cli-plugin-cis-cli#authenticated-origin-pull).
+
+## Example usage
+
+```terraform
+# Change origin auth setting of CIS instance
+
+resource "ibm_cis_orig_auth" "orig_auth_settings" {
+  cis_id                          = data.ibm_cis.cis.id
+  domain_id                       = data.ibm_cis_domain.cis_domain.domain_id
+  certificate                     = EOT<<
+                                  "-----BEGIN CERTIFICATE------ 
+                                   ------END CERTIFICATE-------"
+                                  EOT
+  private_key                     = <<EOT # pragma: whitelist secret
+                                  "-----BEGIN------ 
+                                   ------END--------"  
+                                  EOT
+  hostname                        = "abc.abc.abc.com"
+}
+```
+
+## Argument reference
+
+Review the argument references that you can specify for your resource. 
+
+- `cis_id`                  - (Required, String) The ID of the IBM Cloud Internet Services instance.
+- `domain_id`               - (Required, String) The ID of the domain to change cache settings.
+- `certificate`             - (Required, String) Content of certificate.
+- `private_key`             - (Required, String) Content of private key. # pragma: whitelist secret
+- `hostname`                - (optional, String) Valid host names for host level origin auth processing.
+- `enabled`                 - (optional, Bool)   Default is true, it enables/disables the host and zone level origin auth setting .
+
+
+## Attribute reference
+In addition to all argument reference list, you can access the following attribute reference after your resource is created.
+
+- `id`           - (Computed, String) The record ID. It is a combination of `<auth_id>,<domain_id>,<cis_id>` attributes concatenated with `:`.
+- `updated_at`   - (Computed, String) Time stamp string when Certificate is modified'.
+- `expires_on`   - (Computed, String) Time stamp string when Cerftificate expires on'.
+- `cert_id`      - (Computed, String) Uploaded certificate ID.
+- `status`       - (Computed, String) Origin auth status enbled or not.
+
+
+## Import
+The `ibm_cis_orig_auth` resource can be imported using the ID. The ID is formed from the domain ID of the domain and the CRN concatenated  using a `:` character.
```

Comment:
its a combination of four fields right auth_id:level:domainid:crn..Describe all four

Reply:
Yes, It is changed now.. need to update this also. thanks.